src/sentry_auth_saml2/forms.py

```diff
@@ -23,7 +25,8 @@ def extract_idp_data_from_parsed_data(data):
 
 def process_url(form):
     url = form.cleaned_data['metadata_url']
-    data = OneLogin_Saml2_IdPMetadataParser.parse_remote(url)
+    url_result = fetch_file(url)
```

Comment:
Don't use `fetch_file`. That might be slightly misleading, but this has extra behavior that we don't need or want compared to just `safe_urlopen`.

`fetch_file` adds caching and stuff for event processing pipeline which we don't want here.

Reply:
Ahhh, sounds good, should have looked closer.

Reply:
https://github.com/getsentry/sentry-auth-saml2/pull/19/commits/088259fff08abbcf766963a9ddfbfce7b0a6b9f6